src/en/guide/plugins/creatingAndInstallingPlugins.adoc

```diff
@@ -90,11 +90,20 @@ Adds Quartz job scheduling features
 
 ==== Plugin Configuration
 
+A Grails plugin can be configured in one of the following files:
+
+- `grails-app/conf/plugin.yml`
+- `grails-app/conf/plugin.groovy`
+
+These files will be included in the plugin package while the `application.yml` file is used when the plugin is run as an application. See <<excludedArtefacts>>.
```

Comment:
... while the `application.*` files are only used internally when developing the plugin.